src/Inkluzitron/Modules/HuggingModule.cs

```diff
@@ -52,20 +52,19 @@ public async Task HuggingAsync([Remainder][Name("zpráva")] string message = nul
                 return;
             }
 
-            // iterate over users and find if someone has same name or nickname as message
-            var foundSomeone = false;
+            // iterate over users and find if someone has same name, nickname or ID as message
             foreach (var user in Context.Guild.Users)
             {
-                if (message == user.Nickname || message == user.Username)
+                if (message == user.Nickname || message == user.Username || message == user.Id.ToString())
                 {
-                    foundSomeone = true;
+                    userName = user.Nickname ?? user.Username;
                     await Context.Channel.SendMessageAsync(
-                      $"{Config["Hugging"]} **{message}**"
+                      $"{Config["Hugging"]} **{userName}**"
                     );
+                    return;
                 }
             }
 
-            if (foundSomeone) return;
             await Context.Channel.SendMessageAsync(
               $"Nastala politování hodná situace, nenašel jsem nikoho na ohugování... {Config["PepeHands"]}"
```

Comment:
[politováníhodná](https://ssjc.ujc.cas.cz/search.php?hledej=Hledat&heslo=politov%C3%A1n%C3%ADhodn%C3%BD&sti=EMPTY&where=hesla&hsubstr=no)

Reply:
Jo na to si vlastne uz stezoval Leo. 😆